h4/h4.ino

```diff
@@ -2935,6 +2965,49 @@ bool handleMcode(const String& command) {
   return true;
 }
 
+bool handleChangeToolCommand(String command) {
+  int toolNumber = command.substring(1).toInt();
+  if (toolNumber >= 0 && toolNumber < 4) {
```

Comment:
Should 4 be determined from array length instead of being hardcoded?

Reply:
Yes. Good spot.